docs/dashboards-new/share-dashboards-outside-org.md

```diff
@@ -0,0 +1,111 @@
+
+## Enable the Sumo Logic policy
+
+In this step, you enable sharing of dashboards outside your organization. You must have admin privileges to enable the policy.
+
+1. Go to **Administration** > **Security** > **Policies**.
+1. Select the **Enable** checkbox for **Share Dashboards Outside of the Organization**.
+1. The policy is changed. It can take up to 5 minutes for the policy to go into effect.
+
+## Verify your role allows external sharing
+
+You must have specific role capabilities to share a dashboard with external users.
+
+* **Share Dashboards with the Allowlist.** Share dashboards with users only on the allowlist.
+* **Share Dashboards with the World.** Share dashboards with external users. 
+
+If you do not already have a role that grants you the required capability, an admin user can assign you one. See the [Assign Roles to Users](/docs/manage/users-roles/roles/add-remove-users-role) documentation. Or create a new role as required, see the [Create a New Role](/docs/manage/users-roles/roles/create-manage-roles). 
+
+## Add external users to service allowlist
+
+If you want to share dashboards with specific external users, rather than the world, the service allowlist must be enabled and you must provide the list of IPs or CIDRs that will be able to view dashboards without logging in. For instructions, see [Enable Dashboard Allowlist](/docs/manage/security/create-allowlist-ip-cidr-addresses).
+
+:::note
+Administrator privileges are required to enable the service allowlist.
+:::
+
+## Share the dashboard
+
+1.  In the left navigation pane, right-click the dashboard you want to share, and click **Share** in the context menu. Or, click on the share icon next to the dashboard title. <br/><img src={useBaseUrl('/img/dashboards-new/share-option.png')} alt="share-option" width="400"/> <br/><img src={useBaseUrl('/img/dashboards-new/share-option_2.png')} alt="share-option_2" width="400"/>
+1.  The Share Dashboard dialog appears.
+1.  Click **Get shareable URL** to expand that section of the page. <br/><img src={useBaseUrl('/img/dashboards-new/share-outside.png')} alt="share-outside" width="800"/>
+1.  If you want viewers to see the current time range in the URL, toggle the **Include current time range in the URL** option.
+1.  If you want viewers to see the current variable values in the URL, toggle the **Include current variable values in the URL** option.
+1.  Toggle the **Public Dashboard** on.
+1.  Select **Anyone** or **Allowlist only** as the visibility access, and click **Save**. 
+
+The URL that external users can use to access the dashboard appears under the **Shareable URL** section of the page. 
+
+## Restrictions and limitations for external sharing 
+
+* Sharing outside an organization is always view-only and need auto-refresh enabled. Also, if this is the first time that auto-refresh is being enabled, it may take some time to backfill the data depending on your queries and timeranges. See [About Dashboards](about.md).
```

Comment:
```suggestion
* Sharing outside an organization is always view-only and you must enable auto-refresh. When enabling auto-refresh for the first time, it may take some time to backfill the data, depending on your queries and time ranges. See [About Dashboards](about.md).
```